app/scripts/comp/popup-notifier.js

```diff
@@ -66,7 +71,16 @@ const PopupNotifier = {
                 PopupNotifier.processReturnToApp(toUrl);
             }
         });
-        win.loadURL(url);
+        win.webContents.on('crashed', (e, killed) => {
+            this.logger.debug('crashed', e, killed);
+            setTimeout(PopupNotifier.triggerClosed.bind(PopupNotifier, win), Timeouts.CheckWindowClosed);
+            win = null;
+        });
+        win.webContents.on('did-fail-load', (e, errorCode, errorDescription, validatedUrl, isMainFrame) => {
+            this.logger.debug('did-fail-load', e, errorCode, errorDescription, validatedUrl, isMainFrame);
+            setTimeout(PopupNotifier.triggerClosed.bind(PopupNotifier, win), Timeouts.CheckWindowClosed);
```

Comment:
I'm a bit concerned about this. We're triggering `closed` event here, however the window is not closed. Won't it result in multiple windows being opened because the app thinks there's no active popup?

Reply:
Electron doesn't actually open a window for the cases I've detailed in #456 so I didn't experience any issues with this change. I see you changed to call `win.close`, I think that's good so at least it can free any resources being held.